client/src/services/site.js

```diff
@@ -59,3 +66,65 @@ export const createSite = async (siteJson) => {
   });
   return response;
 };
+
+export const updateSite = async (payload, siteId) => {
+  const response = await fetch(`${API_URL}/api/v1/employer-sites/${siteId}`, {
+    method: 'PATCH',
+    headers: {
+      Authorization: `Bearer ${store.get('TOKEN')}`,
+      Accept: 'application/json',
+      'Content-type': 'application/json',
+    },
+    body: JSON.stringify(payload),
+  });
+  return response;
+};
+
+export const fetchSiteParticipants = async (columnIDs, siteId) => {
+  const response = await fetch(`${API_URL}/api/v1/employer-sites/${siteId}/participants`, {
+    headers: { Authorization: `Bearer ${store.get('TOKEN')}` },
+    method: 'GET',
+  });
+
+  if (response.ok) {
+    const { hired, withdrawn } = await response.json();
+    const hiredParticipants = mapSiteParticipantsDataToRow(hired, columnIDs);
```

Comment:
I'd suggest separating out this logic (separating hired and withdrawn) from the API request itself. Single responsibility principle.

Keep fetchSiteParitcipants to just grabbing the data eand returning it.

Reply:
I think that's reasonable, although I don't have any major issues with the way it is right now either.

Reply:
Updated - Moved the mapping logic to the component. However, regarding inconsistencies, we are mapping the response for `fetchRegionSiteRows` and `fetchSiteRows` in the service file in a similar way before returning to the component.